### PR TITLE
Add canonical links to production logs

### PR DESCRIPTION
**Pull request by brene**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cli-toolchain/usage.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Usage
+canonical: https://docs.amplify.aws/cli/usage/headless
```
Comment (nikhil-dabhade) [inline cli-toolchain/usage.md:3]: Should this be https://docs.amplify.aws/cli/start instead ?
Headless is in advanced section in the new docs.

Comment (brene) [reply cli-toolchain/usage.md]: Good catch!

File: js/start.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+canonical: https://docs.amplify.aws/start
```
Comment (nikhil-dabhade) [inline js/start.md:2]: Should this be https://docs.amplify.aws/start?integration=js

Comment (nikhil-dabhade) [reply js/start.md:2]: Chatted with Rene to clarify this behaviour. We want customers to choose the platform/framework when they select JS on old site. Resolving.

**Pull request merged**